lib/state/ui/reducer.ts

```diff
@@ -70,14 +71,39 @@ const showNoteInfo: A.Reducer<boolean> = (state = false, action) => {
     case 'NOTE_INFO_TOGGLE':
       return !state;
 
-    case 'App.toggleNavigation':
+    case 'NAVIGATION_TOGGLE':
       return false;
 
     default:
       return state;
   }
 };
 
+// @todo
+// const editingTags: A.Reducer<boolean> = (state = false, action) => {
+//   switch (action.type) {
+//     case 'NAVIGATION_TOGGLE':
+//       return false;
+//     default:
+//       return state;
+//   }
+// };
+
+const showNavigation: A.Reducer<boolean> = (state = false, action) => {
+  console.log(action);
+  switch (action.type) {
+    case 'NAVIGATION_TOGGLE':
+      return !state;
+
+    // @todo -- should be implemented with editingTags actually
+    // case 'App.editingTags':
+    // return false;
```

Comment:
when you get to it I'd expect the `@todo` items to appear like so…

```ts
case 'App.selectTag':
case 'App.selectTrash':
case 'App.showAllNotes':
	return false;
```

Reply:
The one I'm stuck on is that it was initially setting `editingTags` to false as well, and... we don't have that in the reducers yet...

Reply:
not sure what you are saying there. if we're getting it from state though and we haven't set it in a reducer it will probably be `undefined` which casts to `false` in boolean conditions